src/main/resources/scripts/rwa_2905_rollback.sql

```diff
@@ -0,0 +1,188 @@
+update cft_task_db.tasks set role_category = null where task_id in (
```

Comment:
Name this file the same as the one making the change and add a `_rollback` suffix.